### PR TITLE
Add tests for Temporal.Duration.p*.abs

### PR DESCRIPTION
**Pull request by FrankYFTang**
#3002  @jugglinmike @ryzokuken @Ms2ger

### CODE REVIEW

File: test/built-ins/Temporal/Duration/prototype/abs/simple.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+description: Temporal.Duration.prototype.abs will return absolute value of the input duration.
+info: |
+  1. Let duration be the this value.
+  2. Perform ? RequireInternalSlot(duration, [[InitializedTemporalDuration]]).
+  3. Return ? CreateTemporalDuration(abs(duration.[[Years]]), abs(duration.[[Months]]), abs(duration.[[Weeks]]), abs(duration.[[Days]]), abs(duration.[[Hours]]), abs(duration.[[Minutes]]), abs(duration.[[Seconds]]), abs(duration.[[Milliseconds]]), abs(duration.[[Microseconds]]), abs(duration.[[Nanoseconds]])).
+
+features: [Temporal]
+---*/
+
+function assertDuration(duration, years, months, weeks, days, hours,
```
Comment (ptomato) [inline test/built-ins/Temporal/Duration/prototype/abs/simple.js:15]: Consider using TemporalHelpers.assertDuration.

**Comment by FrankYFTang** [comment]
Update the test to use temporalHelpers.js and add test for throw. But I cannot find a condiction to cause  CreateTemporalDuration to throw @jugglinmike

**Comment by FrankYFTang** [comment]
see also https://github.com/tc39/proposal-temporal/pull/1698/files

**Comment by FrankYFTang** [comment]
ok, let me just abandon this PR and wait for @ptomato  and @Ms2ger  to move their test from Temporal repo instead.

**Pull request closed without merging**